doc/modules/model_evaluation.rst

```diff
@@ -6,6 +6,113 @@
+target functional, see [Gneiting2009]_.
+For classification **strictly proper scoring rules**, see
+`wikipedia <https://en.wikipedia.org/wiki/Scoring_rule>`_ and [Gneiting2007]_, coincide
+with strictly consistent scoring functions.
+The table further below provides examples.
+One could say that consistent scoring functions act as *truth serum* in that
+they guarantee *"that truth telling [. . .] is an optimal strategy in
+expectation"* [Gneiting2014]_.
+
+Note that for regressors, the prediction is done with :term:`predict` while for
+classifiers it is usually :term:`predict_proba`.
+
+**Decision Making:**
+The most common decisions are done on binary classification tasks, where the result of
+:term:`predict_proba` is turned into a single outcome, e.g., from the predicted
+probability of rain a decision is made on how to act (whether to take mitigating
+measures like an umbrella or not).
+For classifiers, this is what :term:`predict` returns.
+See also :ref:`TunedThresholdClassifierCV`.
+There are many scoring functions which measure different aspects of such a
+decision, most of them are covered with or derived from the :func:`confusion_matrix`.
+
+**List of strictly consistent scoring functions:**
+Here, we list some of the most well known statistical functionals and corresponding
+stritcly consistent scoring functions. Note that the list is not complete and that
+there are more of them.
+For further criteria on how to select a specific one, see [Fissler2022]_.
+
+==================  ===============================================  ===================  ==============================
+functional          scoring or loss function                         response :math:`y`   prediction
+==================  ===============================================  ===================  ==============================
+**Classification**
+mean                :ref:`Brier score <brier_score_loss>`            multi-class          ``predict_proba``
+mean                :ref:`log loss <log_loss>`                       multi-class          ``predict_proba``
+mode                :ref:`zero-one loss <zero_one_loss>` :sup:`1`    multi-class          ``predict``, categorical
+**Regression**
+mean                :ref:`squared error <mean_squared_error>`        all reals             ``predict``, all reals
+mean                :ref:`Poisson deviance <mean_tweedie_deviance>`  non-negative          ``predict``, strictly positive
+mean                :ref:`Gamma deviance <mean_tweedie_deviance>`    strictly positive     ``predict``, strictly positive
+median              :ref:`absolute error <mean_absolute_error>`      all reals             ``predict``, all reals
+quantile            :ref:`pinball loss <pinball_loss>`               all reals             ``predict``, all reals
+mode                no consistent one exists                         reals
+==================  ===============================================  ===================  ==============================
```

Comment:
The intend of this list is pretty unclear to me. 

How does it differ from the list in the section **Common cases: predefined values** that follows this documentation enhancement just a few paragraphs below?

How are users supposed to use it? Maybe an example would help.

Reply:
This table gives you a list of the most used (target) functionals and scoring functions that are strictly consistent for them (first 2 columns).

The table under "Common cases: predefined values" is something completely different. It tells you about the 2nd bullet point under "Scoring API", i.e. the "Scoring parameter": which string argument passed to `cross_val_score(scoring=)` corresponds to which metric (I would not reinvent that design today).

Reply:
> This table gives you a list of the most used (target) functionals and scoring functions that are strictly consistent for them

I see. Actually, it would be nice to have this additional information in the other tables and here focus on teaching people how to use this table, so it becomes clear that it provides additional information on which scoring function should be used in alignment with the goal.

Reply:
It would make the other table very large. This table already contains (with maybe 1-2 exceptions) all scoring functions that are consistent for a functional. All other metrics simply are not consistent for anything.